moduleframework/common.py

```diff
@@ -39,8 +38,35 @@
 import requests
 import warnings
 import ast
+from glob import glob
 from avocado.utils import process
-from moduleframework.mtfexceptions import ModuleFrameworkException, ConfigExc, CmdExc, DefaultConfigExc
+from mtfexceptions import ModuleFrameworkException, ConfigExc, CmdExc, DefaultConfigExc
+from core import DATADIR, is_debug, is_not_silent, print_info, print_debug
+
+
+
+class MTFConfParser(dict):
+    configdir = "mtf.conf.d"
+    source_cfg = os.path.join(*(["/"] + DATADIR + [configdir]))
```

Comment:
Add `/` to DATADIR and remove ["/"].
I would write 
`os.path.join(*(DATADIR, [configdir]))`

What do you want to do with this code? All directories under DATADIR?

Reply:
@phracek this is not possible, it is connected with using this in setup.py (there is just relative path, so I cannot add this to this DATADIR list)